pkg/upgrade/rollout.go

```diff
@@ -0,0 +1,61 @@
+package upgrade
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/threefoldtech/zos/pkg/environment"
+)
+
+const defaultRolloutConfigURL = "https://raw.githubusercontent.com/threefoldtech/zos/development_rollout_update/pkg/upgrade/config.json"
```

Comment:
update the url

Reply:
are we going to maintain it here instead of zos-config repo?https://github.com/threefoldtech/zos-config